falcon/util/misc.py

```diff
@@ -210,3 +214,33 @@ def get_bound_method(obj, method_name):
             raise AttributeError(msg)
 
     return method
+
+
+def get_http_status(status_code):
+    """Gets both the http status code and description from just a code
+
+    Args:
+        status_code: integer or string that can be converted to an integer
+
+    Returns:
+        str: status code e.g. "404 Not Found"
+
+    Raises:
+        ValueError: the value entered could not be converted to an integer
+
+    """
+    # sanitize inputs
+    try:
+        code = decimal.Decimal(str(status_code))  # Decimal can validate values like "401.1"
```

Comment:
If we are expecting either a number or a string, do we need to pass this through `str()`? Also, can we just use `float()` here instead of `Decimal()`, since the value is going to be truncated anyway by `int()`?